polkadot/runtime/parachains/src/paras/mod.rs

```diff
@@ -2337,6 +2484,25 @@ impl<T: Config> Pallet<T> {
 	) -> Option<PvfCheckActiveVoteState<BlockNumberFor<T>>> {
 		PvfActiveVoteMap::<T>::get(code_hash)
 	}
+
+	/// This function checks whether the given `code.hash()` exists in the `AuthorizedCodeHash` map
+	/// of authorized code hashes for a para. If found, it verifies that the associated code
+	/// matches the provided `code`. If the validation is successful, it returns tuple as the
+	/// authorized `ValidationCodeHash` with `expire_at`.
+	pub(crate) fn validate_code_is_authorized(
+		code: &ValidationCode,
+		para: &ParaId,
+	) -> Result<(ValidationCodeHash, BlockNumberFor<T>), Error<T>> {
+		let Some((authorized_code_hash, expire_at)) = AuthorizedCodeHash::<T>::get(para) else {
+			return Err(Error::<T>::NothingAuthorized);
+		};
```

Comment:
```suggestion
		let (authorized_code_hash, expire_at) = AuthorizedCodeHash::<T>::get(para).ok_or(Error::<T>::NothingAuthorized))?
```

just a preference, this is a more common pattern in pallets.